src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__card card" data-qa="card">
+        <div class="card__image-wrapper">
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+        </div>
+
+        <div class="card__description">
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__subtitle">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars stars--4 card__rating-stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__reviews">
+            Reviews: 5
+          </p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">
+            Price:
+          </p>
```

Comment:
semantically `p` means a complete piece of information
But here you have just a part so use `span` or `div`